app/scripts/keyring-controller.js

```diff
@@ -74,31 +78,23 @@ class KeyringController extends EventEmitter {
   // in this class, but will need to be Promisified when we move our
   // persistence to an async model.
   getState () {
-    return Promise.all(this.keyrings.map(this.displayForKeyring))
-    .then((displayKeyrings) => {
-      const state = this.store.getState()
-      // old wallet
-      const wallet = this.configManager.getWallet()
-      return {
-        // computed
-        isInitialized: (!!wallet || !!state.vault),
-        isUnlocked: (!!this.password),
-        keyrings: displayKeyrings,
-        // hard coded
-        keyringTypes: this.keyringTypes.map(krt => krt.type),
-        // memStore
-        identities: this.identities,
-        // configManager
-        seedWords: this.configManager.getSeedWords(),
-        isDisclaimerConfirmed: this.configManager.getConfirmedDisclaimer(),
-        currentFiat: this.configManager.getCurrentFiat(),
-        conversionRate: this.configManager.getConversionRate(),
-        conversionDate: this.configManager.getConversionDate(),
-        // messageManager
-        unconfMsgs: messageManager.unconfirmedMsgs(),
-        messages: messageManager.getMsgList(),
-      }
-    })
+    // old wallet
+    const memState = this.memStore.getState()
+    const result = {
+      // computed
+      isUnlocked: (!!this.password),
+      // memStore
+      keyringTypes: memState.keyringTypes,
+      identities: memState.identities,
+      keyrings: memState.keyrings,
+      // messageManager
+      unconfMsgs: messageManager.unconfirmedMsgs(),
+      messages: messageManager.getMsgList(),
```

Comment:
this will no longer be in keyring controller after the new message manager is merged in. This will most likely cause conflicts